state-sync/storage-service/server/src/moderator.rs

```diff
@@ -137,8 +136,15 @@ impl RequestModerator {
         peer_network_id: &PeerNetworkId,
         request: &StorageServiceRequest,
     ) -> Result<(), Error> {
+        // Time the request validation
+        let _timer = metrics::start_timer(
```

Comment:
the timer is counted when it goes out of scope?

Reply:
Correct 😄